README.md

```diff
@@ -1,6 +1,4 @@
-![alt text][logo]
-
-[logo]: https://github.com/LLNL/merlin/blob/master/docs/images/merlin.png "Merlin logo"
+# Merlin
```

Comment:
We need to point to the raw image:
https://raw.githubusercontent.com/LLNL/merlin/master/docs/images/merlin.png

Reply:
Aha! Thanks

Reply:
@lucpeterson I verified this works. Merging